platform/vechain/transaction_test.go

```diff
@@ -27,15 +27,16 @@ const trxId = `{
 }`
 
 var expectedTransfer = blockatlas.Tx{
-	ID:     "0x702edd54bd4e13e0012798cc8b2dfa52f7150173945103d203fae26b8e3d2ed7",
```

Comment:
Can I kindly ask to create a separate JSON file for tests? According to https://github.com/trustwallet/blockatlas/issues/927 
We need to use separate files for mocks

Reply:
Example 
https://github.com/trustwallet/blockatlas/commit/f3fd50b098e650a1938aadbbda030611bd586cdb